src/App.svelte

```diff
@@ -115,6 +132,31 @@
     scrollBottom = !scrollBottom;
     editor.focus();
   }
+  async function rawGithubGetRequest(url) {
```

Comment:
can re-use the one from `pvmeSettings.js` (fine to export)

Reply:
imported